pyproject.toml

```diff
@@ -644,8 +644,10 @@ disable_error_code = [
 # beginning: stronger typing
 [[tool.mypy.overrides]]
 module = [
+    "src.sentry.tasks.on_demand_metrics",
```

Comment:
Adding these two modules in this block prohibits them from having function signatures without fully typing them.